### DISCUSSION TITLE
Asset creation

### DISCUSSION THREAD

**Discussion by firarasl** [eclipse-tractusx/tractusx-edc#1533] [Q&A]
when creating an asset, Is there a way how to specify access token of baseUrl direcly inside dataAddress here?   
"dataAddress": {
    "@type": "DataAddress",
    "type": "HttpData",
    "baseUrl": "https://mycorp.org/api",
    "oauth2:tokenUrl": "{{ _.url_keycl_backend }}",
    "oauth2:clientId": "{{ _.client_id_backend }}",
    "oauth2:clientSecretKey": "{{ _.sec_name_vault }}",
    "proxyQueryParams": "true",
    "proxyPath": "false",
    "proxyMethod": "true"
  }

**Comment by ndr-brt** [comment]
you can add a `"header:<header_name>": "<header_value>"` property in your `DataAddress`.

Note: please wrap code into triple backtick (```) block, so it will become more readable

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
